app/docs/enterprise/0.31-x/kong-implementation-checklist.md

```diff
@@ -22,7 +22,7 @@ title: Kong Implementation Checklist
 
 ## Production System Requirements
 
-- For Kong prefer CPU optimized instances. We recommend at least 4 CPU cores, 16GB of RAM and 24GB of disk space. Kong performs better on a system with multiple cores. In an AWS environment, we recommend running c 3.2xlarge instances in production that guarantee enough cores (8) and memory (16GB), with a local storage instead of EBS to improve the performance.
+- For Kong prefer CPU optimized instances. We recommend at least 4 CPU cores, 16GB of RAM and 24GB of disk space. Kong performs better on a system with multiple cores. In an AWS environment, we recommend running c4.2xlarge instances in production that guarantee enough cores (8) and memory (16GB), with a local storage instead of EBS to improve the performance.
```

Comment:
Should strike the part about local storage, as current compute-optimized instance sizes are EBS-only.

Reply:
Good catch. Thank you for the careful read and research Travis! Updating.